### DISCUSSION TITLE
DevOps technologies in netbeans

### DISCUSSION THREAD

**Discussion by Richard7111** [apache/netbeans#4978] [Q&A]
Hello,
I am a student of a technical university and one of my assignments is to analyse DevOps technologies of open source projects.
I see that netbeans is using Travis CI, Jenkins and GitHub actions.
Do I understand correctly that Jenkins is used for building the IDE for Linux or Windows and the others are for testing? Or am I overlooking something?

**Comment by mbien** [comment]
some replies on the dev list:
https://lists.apache.org/thread/pdop7qympkshrolxhcx304vf7q499spn
